pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -1184,8 +1184,9 @@ public static void updateKit(String dsmKitId, Shipment participantShipment, Ship
         if (results.resultException != null) {
             logger.error("Error updating kit w/ dsm_kit_id " + dsmKitId, results.resultException);
         } else {
-            logger.info("Updated kit w/ dsm_kit_id " + dsmKitId, results.resultException);
-
+            logger.info("Updated kit w/ dsm_kit_id " + dsmKitId);
+        }
```

Comment:
So there's a case where the `setDsmKitId` call fails but we still want to continue?
If so, maybe an explanation?

Reply:
Oh I see what you mean, great catch! I will fix this